cmd/buildah/run.go

```diff
@@ -67,7 +68,9 @@ func init() {
 	// TODO add-third alias for tty
 	flags.BoolVarP(&opts.terminal, "terminal", "t", false, "allocate a pseudo-TTY in the container")
 	flags.BoolVar(&opts.terminal, "tty", false, "allocate a pseudo-TTY in the container")
-	flags.MarkHidden("tty")
+	if err := flags.MarkHidden("tty"); err != nil {
+		panic(fmt.Sprintf("error marking tty as hidden: %v", err))
```

Comment:
This should explicitly mention that it's about the "tty flag" to be less confusing.

Reply:
Good catch!